### ISSUE TITLE
Error in javascript tracker loading: bad sanitization`

### ISSUE THREAD

**Issue by mbolis** [padams/Open-Web-Analytics#381]
Javascript code in the page is failing to load, giving me this error in the console (Firefox):

> SyntaxError: expected expression, got '&'

The problem is the `$cmds` entries get output with `$sanitize = true` in modules/base/templates/js_log_tag.tpl, so the output is:

```
owa_cmds.push([&#039;trackPageView&#039;]);
owa_cmds.push([&#039;trackClicks&#039;]);
```

**Comment by davetbo** [comment]
I'm getting this issue as well on 1.6.2.  Please help! :)

**Comment by yrammos** [comment]
Same here.

**Comment by yrammos** [comment]
UPDATE: This seems resolved in 73bd2482ceb6d8e90c04bac093b83468e4c21109. See also  https://github.com/padams/Open-Web-Analytics/issues/384#issuecomment-471310813.

**Comment by padams** [comment]
closing
